### ISSUE TITLE
Identify list of tools for the GW community

### ISSUE THREAD

**Issue by david-lippert** [gwu-libraries/OSPO#4]
Identify and create list of tools, educational resources, and tools specific to help manage the OSPO.
add to dynamic doc here: https://gwu.box.com/s/1kvi6b18900c5fh7coaku8dpxgmt3y1e

**Comment by labarba** [comment]
I noticed you have listed Mattermost in the Box doc. At the same time, I noticed that in the Slack space for "GW Coders" several channels appear empty, but could have older posts that are not visible because they are using the free tier. It makes me think that we could serve the community well by offering a self-hosted Mattermost server to replace free-tier Slack... I bet a lot of student groups would benefit. (Maybe this could be a job for the OSPO dev.)

**Comment by david-lippert** [comment]
@labarba I agree.  I think there are many considerations regarding switching from slack to mattermost, but I will explore with IT and get their thoughts.
Here are some questions I have:
- what role if any would the OSPO have in maintaining mattermost?
- how many licensed instances of slack exist at GW?
- how many free instances of slack exist at GW?
- what would the cost be of hosting and moving all of the slack instances to mattermost?
- how does the capability compare?  would slack users lose any capability?
- are there other benefits to self hosting with mattermost (cost, security)?

**Comment by labarba** [comment]
From the responses I got on the GW Coders Slack group from John Helveston and Ryan Watkins, it sounds like the idea of a Mattermost server would be a bust. So let's can that one!

**Comment by david-lippert** [comment]
I agree.  The conversation is interesting and I wonder if there might be work to be done (perhaps across OSPOs) to set standards or recommendations to discourage companies from applying these vendor lock-in business models.  Perhaps a set of guidelines around open practices to encourage APIs, multiple integration points, using standard protocols and formats such that the best tool would win rather than whatever tool currently has the marketshare lead.

Does OSI have definitions that extend to "friendly" to open source for companies that are not open source, but adjacent to open source?

**Comment by labarba** [comment]
Relatedly, at the CURIOSS event in Pittsburgh, there was an "infrastructure" break-out group where folks discussed GitHub questioning whether we should rely on it so much. In the case of GitHub to be honest I think we're in too deep!
